SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Modules/SalemoveSDK.swiftmodule/arm64-apple-ios.abi.json

```diff
@@ -35459,7 +35459,7 @@
       "kind": "StringLiteral",
       "offset": 433,
       "length": 9,
-      "value": "\"0.35.13\""
+      "value": "\"0.35.14\""
```

Comment:
Is it ok that new version is different there and in SalemoveSDK.podspec and Package.swift files?